ranger-http/src/main/java/io/appform/ranger/http/config/HttpClientConfig.java

```diff
@@ -34,4 +34,5 @@ public class HttpClientConfig {
     long connectionTimeoutMs;
     long operationTimeoutMs;
     long refreshIntervalMillis;
+    boolean replicationSource;
```

Comment:
Would you want to set the default value to this, say to false, in the config model itself? For the rest of the timeouts, RangerHttpUtils handles this I think

Reply:
Also possibly @JsonIgnoreProperties(ignoreUnknown = true)?